ansible/configs/ocp4-cluster/default_vars_osp.yml

```diff
@@ -33,20 +32,24 @@ ansible_user: cloud-user
 remote_user: cloud-user
 
 # The base domain
-ocp4_base_domain: "{{ cluster_dns_zone }}"
+# osp_cluster_dns_zone needs to come from secrets
+# subdomain_base_suffix: "{{ osp_cluster_dns_zone }}"
+ocp4_base_domain: "{{ osp_cluster_dns_zone }}"
 
 # The dynamic DNS server you will add entries to.
 # NOTE: This is only applicable when {{ use_dynamic_dns}} is true
-osp_cluster_dns_server: ddns01.opentlc.com
-
-# Whether to wait for an ack from the DNS servers before continuing
-wait_for_dns: true
+# This needs to come from secrets
+# osp_cluster_dns_server: ddns01.opentlc.com
```

Comment:
I think this was moved to secrets and can properly be something like `FROMSECRET` here?

Reply:
Good point.